src/Spryker/Client/RabbitMq/Model/Helper/QueueEstablishmentHelper.php

```diff
@@ -17,9 +17,9 @@ class QueueEstablishmentHelper implements QueueEstablishmentHelperInterface
      * @param \PhpAmqpLib\Channel\AMQPChannel $channel
      * @param \Generated\Shared\Transfer\RabbitMqOptionTransfer $queueOptionTransfer
      *
-     * @return array
+     * @return mixed[]
```

Comment:
Is there a difference?

Reply:
I don't think so :)

Reply:
Should I change it back?

Reply:
We usually don't use mixed[] as it provides no additional value.